functions/billing/index.js

```diff
@@ -62,7 +62,7 @@ exports.stopBilling = async (pubsubEvent, context) => {
     return `No project specified`;
   }
 
-  await _setAuthCredential();
+  _setAuthCredential();
```

Comment:
Well, if you're not going to await an async function, you're just asking for issues.

I think you mean to remove `async` from `_setAuthCredential` too.

Reply:
Fixed.